backend/config/routes.rb

```diff
@@ -18,7 +18,9 @@
         end
         post 'studies/:id/update_status', to: 'studies#update_status'
 
-        resources :analysis, except: [:destroy]
+        resources :analysis, except: [:destroy] #  do
+        #   resources :runs, only: [:create], shallow: true
```

Comment:
todo

Reply:
hm, I think that can actually be removed.  I'll do so.